packaging/installer/dependencies/arch.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+# Package tree used for installing netdata on distribution:
+# << ArchLinux >>
```

Comment:
This one will also end up used (without any need for changes) on Manjaro, and possibly other Arch derivatives.